apps/token-manager/app/src/App.js

```diff
@@ -86,100 +98,151 @@ class App extends React.PureComponent {
   render() {
     const {
       appStateReady,
+      connectedAccount,
       groupMode,
       holders,
       isSyncing,
+      layoutName,
       maxAccountTokens,
       numData,
+      requestMenu,
+      theme,
       tokenAddress,
       tokenDecimalsBase,
       tokenName,
       tokenSupply,
       tokenSymbol,
       tokenTransfersEnabled,
-      connectedAccount,
-      requestMenu,
     } = this.props
+
     const { assignTokensConfig, sidepanelOpened } = this.state
 
     return (
-      <Main assetsUrl="./aragon-ui">
-        <div css="min-width: 320px">
-          <IdentityProvider
-            onResolve={this.handleResolveLocalIdentity}
-            onShowLocalIdentityModal={this.handleShowLocalIdentityModal}
-          >
-            <SyncIndicator visible={isSyncing} />
-            <AppLayout
```

Comment:
Should we also remove `AppLayout.js` and this import?

Reply:
Yes!